src/odemis/util/concurrent.py

```
@@ -0,0 +1,430 @@
+    """
+    if kwargs is None:
+        kwargs = {}
+    thread = threading.Thread(target=bindFuture,
+                              name="Future runner",
+                              args=(future, fn),
+                              kwargs={"args": args, "kwargs": kwargs})
+    thread.start()
+    return thread
+
+
+def bindFuture(future, fn: Callable, args: tuple = (), kwargs: dict = None) -> None:
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)